seqr/utils/social_auth_pipeline_tests.py

```diff
@@ -0,0 +1,26 @@
+import mock
+import responses
+from unittest import TestCase
+
+from social_core.backends.google import GoogleOAuth2
+from seqr.utils.social_auth_pipeline import validate_anvil_registration
+from settings import TERRA_API_ROOT_URL
+
+
+class SocialAuthPipelineTest(TestCase):
+
+    @responses.activate
+    @mock.patch('seqr.utils.social_auth_pipeline.logger')
+    def test_validate_anvil_registration(self, mock_logger):
+        url = TERRA_API_ROOT_URL + 'register'
+        responses.add(responses.GET, url, status = 404)
+        r = validate_anvil_registration(GoogleOAuth2(), {'access_token': '', 'email': 'test@seqr.org'})
```

Comment:
is this ACTUALLY calling google? or is the responses handling that?

Reply:
`responses` handle it.